web/views/FcRequestStudyBulkView.vue

```diff
@@ -0,0 +1,292 @@
+<template>
+  <section class="fc-request-study-bulk-view d-flex flex-column fill-height">
+    <v-row
+      class="align-center d-flex flex-grow-0 flex-shrink-0 px-3 py-2 shading"
+      no-gutters>
+      <v-col cols="2">
+        <FcButton
+          type="secondary"
+          @click="actionNavigateBack">
+          <v-icon left>mdi-chevron-left</v-icon>
+          {{labelNavigateBack}}
+        </FcButton>
+      </v-col>
+      <v-col class="text-center" cols="8">
+        <h1 class="headline">
+          <span>
+            Bulk Request #{{$route.params.id}}:
+          </span>
+          <v-progress-circular
+            v-if="loading"
+            color="primary"
+            indeterminate
+            :size="20"
+            :width="2" />
+          <span
+            v-else
+            class="font-weight-regular">
+            {{studyRequestBulk.name}}
+          </span>
+        </h1>
+      </v-col>
+      <v-col class="text-right" cols="2">
+        <FcButton
+          v-if="studyRequestBulk !== null && canEdit && bulkStatus.editable"
+          :disabled="loading"
+          type="secondary"
+          @click="actionEdit">
+          <v-icon color="primary" left>mdi-pencil</v-icon> Edit
+        </FcButton>
+      </v-col>
+    </v-row>
+
+    <v-divider></v-divider>
+
+    <section class="flex-grow-1 flex-shrink-1 overflow-y-auto">
+      <v-progress-linear
+        v-if="loading"
+        indeterminate />
+      <div v-else>
+        <v-row
+          class="mb-6"
+          no-gutters>
+          <v-col class="mt-6 px-5" cols="12">
+            <h2 class="display-3 mb-4">
+              {{studyRequestBulk.name}}
+            </h2>
+            <FcStatusStudyRequestBulk
+              class="mt-2"
+              :study-request-bulk="studyRequestBulk"
+              :study-request-changes="studyRequestChanges" />
+          </v-col>
+          <v-col cols="6">
+            <FcSummaryStudyRequestBulk
+              class="mx-5"
+              :study-request-bulk="studyRequestBulk"
+              :study-request-users="studyRequestUsers"
+              style="min-height: 320px;" />
```

Comment:
can we remove this and stick a `min-height` on `<FcPaneMap>`'s root element?  that way we can be assured that all maps will have a minimum height.